src/web/shared/Utilities.tsx

```diff
@@ -150,6 +150,11 @@ function handleAccordionDropdownClick(event: Event, targetClassName: string, par
 
 export function adjustAccordionScrollPosition(parentContainerId: string, dropdownContainer: HTMLElement) {
     const parentContainer = document.getElementById(parentContainerId)
+
```

Comment:
Since we are updating this function can we update the comment to use the correct /**/